sample-compose/domain/src/main/java/co/nimblehq/sample/compose/domain/model/Model.kt

```diff
@@ -1,5 +1,6 @@
 package co.nimblehq.sample.compose.domain.model
 
 data class Model(
-    val id: Int?
+    val id: Int?,
+    val userName: String?,
```

Comment:
Same here.
```suggestion
    val username: String?,
```

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/480/commits/991c23356f8d3add114f86ff6832d905e51efa07